lib/companies_web/templates/company/recent.html.eex

```diff
@@ -42,6 +42,9 @@
           </div>
         <% end %>
       </div> <!-- /columns -->
+      <div class="container has-text-centered">
+        <%= link gettext("Browse full list"), to: Routes.company_path(@conn, :index), class: "button is-link" %>
```

Comment:
Perhaps you will tell more correct text in English.

Reply:
This is fine. You could also say: "Browse All Companies" or  "Browse Companies"